semantic/src/site/globals/site.overrides

```diff
@@ -862,6 +862,17 @@ a:hover {
   }
 }
 
+.history-container {
```

Comment:
Just wondering why this needs its own class -- the container looks exactly the same as the main response-container.

Reply:
The main difference is that it doesn't have the `flex-row` because elements display in a horizontalBox.